### ISSUE TITLE
Login failed on live ios paypal sdk

### ISSUE THREAD

**Issue by mingtai511** [paypal/PayPal-iOS-SDK#569]
### General information

* SDK/Library version: iOS SDK 2.18.0
* Environment: Production
* `PayPal-Debug-ID` values: a2110099ab065
* iOS Version and Device: iOS 11, iphone8plus simulater and iphone6 real phone
* Integration type and version: Cocoapod

### Issue description

I integrated paypal sdk to my ios app
After integrated it, I tested pay on Sandbox mode, working well
But when i test pay on live mode, i can 't login with my personal paypal account
So I tested sample app in SDK
But i got same result
In other words, I can login and pay on sandbox in sample app, but i can not login paypal on live mode
I attatched log screenshot
http://prntscr.com/i2p9q5
Please check and let me know
Maybe I mistake some setting on developer.paypal.com?
http://prntscr.com/i2pcpk
It's my setting on developer.paypal.com
Please let me know what should i do more?
Thanks

**Comment by pp-randy** [comment]
That looks like a username and password issue.  Are you able to log into PayPal.com with that same account?

**Comment by mingtai511** [comment]
Yes

**Comment by mingtai511** [comment]
http://prntscr.com/i2pmf4
Maybe my business account should be pro account?

**Comment by pp-randy** [comment]
Can you log in with a different account or are all logins broken?

**Comment by pp-randy** [comment]
It looks like you are using a simulator and not using an actual phone.  Does it work if you push to a phone?

**Comment by mingtai511** [comment]
I tested it on real phone, too
But logged in failed

**Comment by pp-randy** [comment]
Try a different PayPal account for login. Create a new one if you don't have one and run a test.  

If that works, I encourage you to follow up with Customer Support for your other account at https://www.paypal.com/help.

**Comment by mingtai511** [comment]
How about it?
http://prntscr.com/i2pmf4
Maybe my business account should be pro account?

**Comment by pp-randy** [comment]
That is only needed if you want to process credit cards; however, processing credit cards is discourage with this SDK now. 

Since your integration works in Sandbox, this will most likely take access to live information.  You will need contact Technical Support if my above suggestion about testing a different account also fails.  You can contact PayPal Technical Support here: https://www.paypal-techsupport.com/app/ask.

**Comment by mingtai511** [comment]
OK, thanks

**Comment by pp-randy** [comment]
The only other thing to check for your settings you show here https://prnt.sc/i2pcpk, make sure that is set on the Live tab.

**Comment by mingtai511** [comment]
Yes
It's live setting

**Comment by xiaoleih41** [comment]
Can you double check the app secret matches to the one shown on your developer.com?

**Comment by xiaoleih41** [comment]
Close this issue due to inactivity. Please resubmit if the issue is not solved.
